jina/parser.py

```diff
@@ -305,8 +305,6 @@ def set_pea_parser(parser=None):
     gp3.add_argument('--dump-interval', type=int, default=240,
                      help='serialize the model in the pod every n seconds if model changes. '
                           '-1 means --read-only. ')
-    gp3.add_argument('--exit-no-dump', action='store_true', default=False,
```

Comment:
Why? Breaking change?

Reply:
breaking change yes. I removed because I have never seen it used and complicates the closing logic. I have the feeling that is weird to have read_only and exit_no_dump to work on different directions